packages/wallet/src/nodejs/index.ts

```diff
@@ -95,13 +95,13 @@ export class SingleAddressWallet implements WalletAPI {
                                 , O.getOrElse(() => 0n))))
 
 
-    public tokenBalance : (token : Token) => TE.TaskEither<Error,bigint>
-        = (token) =>
+    public tokenBalance : (assetId : AssetId) => TE.TaskEither<Error,bigint>
```

Comment:
I don't understand this change, are you changing Token for Asset in the codebase? I would avoid that as in all the other repos Token is used. Asset is informally the Token+Value combination

Reply:
Yes, I'm aligning with what Jamie has done in the runtime